test/unit/notices_Do.js

```diff
@@ -84,53 +61,13 @@ var coreNotice1 = {
 
 describe('Notices Do', function() {
     describe('#DoNotice', function() {
-        var v1notice, v2notice;
+        var /*v1notice,*/ v2notice;
 
         beforeEach(function() {
-            v1notice = JSON.parse(noticeExampleV1);
+            //v1notice = JSON.parse(noticeExampleV1);
```

Comment:
Leftover to be deleted?

Reply:
done in 93cadb1a00a19a60be02cb02d173cac775f6a3e7